src/StakeVault.sol

```diff
@@ -202,6 +202,13 @@ contract StakeVault is IStakeVault, Initializable, OwnableUpgradeable {
         _withdraw(_token, _amount, _destination);
     }
 
+    /**
+     * @notice Compounds the MPs tp increase the vault's share of the rewards.
+     */
+    function compound() external onlyOwner onlyTrustedStakeManager {
```

Comment:
I think we can allow everyone to claim for everyone no?
This could even later allow status to do the MP staking for all users if they wanted to.